src/eko/output/struct.py

```diff
@@ -0,0 +1,502 @@
+# -*- coding: utf-8 -*-
+import contextlib
+import io
+import logging
+import os
+import pathlib
+import shutil
+import tarfile
+import tempfile
+import time
+import typing
+from dataclasses import dataclass, fields
+from typing import BinaryIO, Dict, Literal, Optional, Tuple
+
+import lz4.frame
+import numpy as np
+import numpy.lib.npyio as npyio
+import yaml
+
+from .. import basis_rotation as br
+from .. import interpolation
+from .. import version as vmod
+
+logger = logging.getLogger(__name__)
+
+PathLike = typing.Union[str, os.PathLike]
```

Comment:
is there no standard type? this should appear all over the place ...

Reply:
I don't know, I thought `os.PathLike` was the standard type, but most likely the static analyzer complained, I will try again without and check for the issues.

Reply:
Solved in a621d847da996f9498c2a32bf0658d4d8aa28432